docs/templates/modules/hub.md

```diff
@@ -71,6 +76,8 @@ On this page:
 >
 > - pipelines – Create the following nested pipelines:
 >   - **ms-cm-exports_Transform** – Converts the CSV export to parquet and moves it to the **ingestion** container (same path), overwriting the existing contents for that month.
+>   - **ms-cm-exports_Retention** – Clean up files based on the data retention settings.
+>   - **ingestion_Retention** – Clean up files based on the data retention settings.
```

Comment:
@MSBrett I added pipelines for data retention since I don't think we can reuse storage retention for month-based retention. Let me know your thoughts. We could also just round up the number of days in storage retention to avoid extra pipeline run costs.

Reply:
@tsilvers-ms / @ankur-ms, I'd love your thoughts as well!